_assets/master.css

```diff
@@ -1775,23 +1775,37 @@ h4 {
   position: absolute;
   left: -0.4em;
 }
-.post h3,
-.post h4 {
-  font-family: CircularStdBook, CircularStdBold, sans-serif;
-  font-size: 2vw;
-  margin-top: 2rem;
+.postContent h1 {
+  font-size: 1.5rem;
+  line-height: 200%;
+  margin: 1rem 0;
+}
+@media (min-width: 62em) {
+  .postContent h1 {
+    font-size: 2rem;
+  }
+}
+.postContent h2 {
+  font-size: 1.7rem;
+}
+@media (min-width: 62em) {
+  .postContent h2 {
+    font-size: 2rem;
+  }
+}
+.postContent h3 {
+  font-size: 1rem;
 }
-.post strong,
-.post em {
+.postContent em {
   font-family: CircularStdBook, CircularStdBold, sans-serif;
```

Comment:
Why this font stack? First bold then... fall back to bold? Please use the variables in vars.styl if you can.